.github/workflows/ci.yml

```diff
@@ -20,20 +20,18 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-      with:
-          submodules: true
 
     - name: Install Dependencies
       run: |
         apt update
-        apt install -y libgala-dev libgee-0.8-dev libglib2.0-dev libgranite-dev libgtk-3-dev libhandy-1-dev \
-        libdbus-glib-1-dev libwnck-3-dev libgtop2-dev libwingpanel-3.0-dev libudisks2-dev \
+        apt install -y libgala-dev libgee-0.8-dev libglib2.0-dev libgranite-dev libgtk-4-dev libadwaita-1-dev \
+        libdbus-glib-1-dev libgtop2-dev libwingpanel-3.0-dev libudisks2-dev \
         libxnvctrl0 libxnvctrl-dev libcurl4-gnutls-dev libflatpak-dev libjson-glib-dev \
-        liblivechart-1-dev libpci-dev \
+        liblivechart-2-dev libpci-dev \
         meson valac sassc git
     - name: Build
       run: |
-        meson setup -Dindicator-wingpanel=enabled build
+        meson setup -Dindicator-wingpanel=disabled build
```

Comment:
We build the indicator in its own separate build file so this shouldn't need to be touched